test/src/kbasesearchengine/test/main/SearchMethodsTest.java

```diff
@@ -669,4 +673,175 @@ private void compare(final SortingRule got, final SortingRule expected) {
                 is(expected.getIsObjectProperty() == null ? 1L : expected.getIsObjectProperty()));
     }
 
+    private SearchInterface setUpTestAccessFilter () throws Exception{
+        final AccessGroupProvider agp = mock(AccessGroupProvider.class);
+        final IndexingStorage is = mock(IndexingStorage.class);
+        final TypeStorage ts = mock(TypeStorage.class);
+
+        final Set<String> admins = new HashSet<>(Arrays.asList("auser"));
+        final SearchInterface sm = new SearchMethods(agp, is, ts, admins);
+
+        final List<Integer> adminList = new ArrayList<>();
+        adminList.add(2);
+        final Map<String, Integer> privateData = new HashMap<>();
+        privateData.put("test", 1);
+
+        final Map<String, Integer> publicData = new HashMap<>();
+        publicData.put("notdefault", 1);
+
+        final Map<String, Integer> allData = new HashMap<>();
+        allData.put("test", 1);
+        allData.put("notdefault", 1);
+
+        when(agp.findAccessGroupIds("auser"))
+                .thenReturn(adminList);
+
+        when(is.searchTypes( kbasesearchengine.search.MatchFilter.getBuilder().build(),
+                new kbasesearchengine.search.AccessFilter()
+                        .withAccessGroups(new LinkedHashSet(adminList))
+                        .withAdmin(true)
```

Comment:
why is admin true for these tests?

Reply:
b/c I made auser an admin

Reply:
But why did you do that?

Reply:
b/c at one point I thought i needed to test that too. I guess i can remove it